Dockerfile

```diff
@@ -25,7 +25,7 @@ FROM builder AS evm-loader-builder
 COPY ./evm_loader/ /opt/evm_loader/
 WORKDIR /opt/evm_loader/program
 RUN cargo build-bpf --features no-logs
-RUN cd ../cli && cargo build --release
+# RUN cd ../cli && cargo build --release
```

Comment:
neon-cli have conflicts with  solana-sdk 1.6.9 needs to be updated

Reply:
neon-cli used in tests and proxy so it needs to be repaired.